LICENSE

```diff
@@ -1,94 +1,203 @@
-License for everything not in third_party and not otherwise marked:
```

Comment:
Pls add readme files

Reply:
There is a README file for the project, is there something else you are talking about..